examples/genomicsdb_query

```diff
@@ -529,6 +580,40 @@ def main():
     if len(configs) == 0:
         print("Nothing to process!!. Check output for possible errors")
         sys.exit(1)
+
+    # Check if there is room for row_tuples to be parallelized
+    chunk_size = int(args.chunk_size)
+    if len(configs) < args.nproc and chunk_size > 1:
+        if row_tuples is None:
```

Comment:
Should we consider only doing the splitting by row tuples if all samples are being queried? I'm worried about the case where only a few random (but disjoint) samples are queried and that could lead to us creating a whole bunch of configs where each config is only querying a single sample

Reply:
Sure, you have a point and we could do the splitting only if all samples are being queried.